client-tools/Restore/RestoreFeature.cpp

```diff
@@ -1670,9 +1670,8 @@ RestoreFeature::RestoreFeature(Server& server, int& exitCode)
   using arangodb::basics::FileUtils::buildFilename;
   using arangodb::basics::FileUtils::currentDirectory;
   _options.inputPath = buildFilename(currentDirectory().result(), "dump");
-  _options.threadCount =
-      std::max(uint32_t(_options.threadCount),
-               static_cast<uint32_t>(NumberOfCores::getValue()));
+  // Default ThreadCount 4
+  _options.threadCount = std::max(uint32_t(_options.threadCount), 4);
```

Comment:
Same here with `client-tools/Restore/RestoreFeature.h` and `struct Options` there.